versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -86,5 +86,8 @@ spec:
       fields:
         description: Sub-fields, when type is group
         $ref: "#"     # JSON-schema syntax for pointing to the root of the schema
+      dynamic_template:
```

Comment:
nit: could you please put it before `fields`? Let's leave the subgroup of fields as the last item here.